src/components/inputs/rich-text-input/button.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
From experience navigating the MC a lot I would suggest calling the file `rich-text-input-button` and the component `RichTextInputButton`. It really helps when just jumping files with a fuzzy finder. Sorry for a naming nitpick but it's more of a general naming pattern also I suppose.